azure_functions_worker/dispatcher.py

```diff
@@ -346,13 +344,11 @@ async def _handle__function_load_request(self, req):
 
     async def _handle__invocation_request(self, req):
         invoc_request = req.invocation_request
-
         invocation_id = invoc_request.invocation_id
         function_id = invoc_request.function_id
-        trace_context = bindings.TraceContext(
-            invoc_request.trace_context.trace_parent,
-            invoc_request.trace_context.trace_state,
-            invoc_request.trace_context.attributes)
+        trace_context, retry_context = \
+            self._get_trace_and_retry_context(invoc_request)
```

Comment:
You can name it as `get_context` as in the future there might be more contexts might come and we don't want to change this again.